_includes/rest/objects.md

```diff
@@ -165,6 +165,33 @@ print result
 </code></pre>
 </div>
 
+When using a MongoDB replica set, you can use the `readPreference` option to choose from which replica the object will be retrieved. You can also use the `includeReadPreference` option to choose from which replica the included pointers will be retrieved. The possible values for both options are `PRIMARY` (default), `PRIMARY_PREFERRED`, `SECONDARY`, `SECONDARY_PREFERRED`, or `NEAREST`. If the `includeReadPreference` option is not set, the same replica chosen for `readPreference` will be also used for the includes.
+
```

Comment:
Same rule applies to subqueryReadPreference

Reply:
When getting an object, the `subqueryReadPreference` does not make sense. I think you can even pass it, but it will never do any thing.

Reply:
Do you agree to not mention it?

Reply:
I agree